com.trollworks.gcs/src/com/trollworks/gcs/weapon/WeaponDamage.java

```diff
@@ -272,10 +295,20 @@ public String getResolvedDamage(StringBuilder toolTip) {
                         base.multiply(advantage.getLevels());
                     }
                 }
+                //Phoenix D3
+                double percent = (double) 0;
```

Comment:
You should use 0.0 instead of a cast here.

Reply:
Thanks, will do. I have no formal java training so thanks again for being patient with me. This change will be committed soon™.